netlify.toml

```diff
@@ -287,7 +287,8 @@
 
 # Handle 404s in our app, SPA redirect config
 # SHOULD ALWAYS BE LAST!
+
 [[redirects]]
   from = "/*"
   to = "/"
-  status = 200
\ No newline at end of file
+  status = 200
```

Comment:
Testing the prior netlify build it looks like 404s were broken already It seems like this last part would be 

```
[[redirects]]
  from = "/*"
  to = "/404"
  status = 404
```